README.md

```diff
@@ -45,14 +45,16 @@ Connect the ribbon from the epaper display to the extension.  To do this you wil
 
 ## Setup dependencies
 
-    sudo apt install git ttf-wqy-zenhei ttf-wqy-microhei python3 python3-pip python-imaging libopenjp2-7-dev libjpeg8-dev inkscape figlet wiringpi
-    sudo pip3 install python-dateutil astral spidev RPi.GPIO Pillow  # Pillow took multiple attempts to install as it's always missing dependencies
+    sudo apt update && sudo apt upgrade  
+    sudo apt install git fonts-wqy-zenhei fonts-wqy-microhei python3 python3-pip inkscape figlet pigpio python3-pigpio  
+    sudo pip3 install python-dateutil astral spidev RPi.GPIO Pillow  
```

Comment:
Wiringpi is no longer available, replacing it with pigpio.  

Also it doesn't look like the libjpeg and python-imaging packages are needed anymore, I just install Pillow and that seems to work just fine.